data/legend/metadata/datasets/cal_groupings.yaml

```
@@ -0,0 +1,3 @@
+default:
+  calgroup001a:
+    p02: r000...r007
```

This also breaks our code
```suggestion
    p02: r000..r007
```

---

sure that would be a simple change our end